frontend/js/actions/setBattle.js

```diff
@@ -0,0 +1,12 @@
+import { getTeamData } from 'utils/api';
```

Comment:
I don't like the names of actions: `getBattleList`, `setBattle`, `setUser`. let's keep the consistency and select one patter